src/firebolt_ingest/table_model.py

```diff
@@ -129,13 +129,23 @@ class Table(BaseModel, YamlModelMixin):
     primary_index: conlist(str, min_items=1)  # type: ignore
     partitions: List[Partition] = []
     file_type: str
-    object_pattern: conlist(str, min_items=1)  # type: ignore
+    object_pattern: str
     compression: Optional[str] = None
     csv_skip_header_row: Optional[bool] = None
     json_parse_as_text: Optional[bool] = None
     sync_mode: Optional[str] = None
     s3_url: Optional[str] = Field(regex=r"^s3:\/\/[a-z0-9-]{1,64}\/[a-zA-Z0-9-_.\/]*")
 
+    @root_validator
+    def object_pattern_validator(cls, values: dict) -> dict:
```

Comment:
Is this required? Doesn't pydantic check this automatically since object_pattern is defined as `str` and not `Optional[str]`?

Reply:
yeah, it checks if object_pattern exists in yaml and is not None, but it will pass if I add it like this object_pattern: "".
Then firebolt fails with Line 4, Column 18: Invalid SQL: syntax error, unexpected EMPTY_IDENTIFIER
OBJECT_PATTERN = "" if object_pattern is an empty string